docs/Antora/modules/howto_guides/pages/howto_configure_instance.adoc

```diff
@@ -40,6 +40,9 @@ instance = instance_builder.build()
 ----
 ====
 
+== Configuring openDAQ(TM) Instance from external providers
+Instance builder supports configuring from external providers like JSON file, environment variables, or command line arguments. More information about how to use providers see xref:howto_configure_instance_providers.adoc[Configure Instance providers] guide.
```

Comment:
Is this correct expression "Instance provider"? I assume we are talking about "Configuration provider"